packages/ui/src/FormErrorsDisplay/FormErrorsDisplay.tsx

```diff
@@ -1,9 +1,9 @@
-import AlertFormError from '@/components/AlertFormError'
+import AlertFormError from '../../../../apps/main/src/components/AlertFormError'
```

Comment:
It shouldn't be like this

Reply:
yeah for sure, this is unused now, I'll remove